posthog/async_migrations/test/test_0002_events_sample_by.py

```diff
@@ -114,3 +129,32 @@ def test_run_migration_in_full(self):
 
         self.assertEqual(events_count_res[0][0], 5)
         self.assertEqual(backup_events_count_res[0][0], 5)
+
+    @pytest.mark.ee
+    @mock.patch("posthog.async_migrations.utils.execute_op_clickhouse", side_effect=mocked_execute_op_clickhouse)
+    def test_run_migration_in_full_optimize_fails(self, _):
+        from ee.clickhouse.client import sync_execute
+
+        migration_successful = start_async_migration(MIGRATION_NAME)
+        sm = AsyncMigration.objects.get(name=MIGRATION_NAME)
```

Comment:
Nit: add a blank line before this.

Tests are usually structured as SETUP - METHOD_UNDER_TEST - ASSERT. this is part of the assert code, hence should be visually distinct from the line above.